### DISCUSSION TITLE
Is it possible to restart zephyr os from baremetal side?

### DISCUSSION THREAD

**Discussion by pmittireddy** [bao-project/bao-hypervisor#194] [General]
for dual guest (zephyr+baremetal), is there a way where we could restart zephyr vm from baremetal vm?

**Comment by josecm** [comment]
No. At the moment it is not possible to restart a VM. There is a wip [PR](https://github.com/bao-project/bao-hypervisor/pull/157) that allows this. But it only allows a VM to restart itself.

**Comment by pmittireddy** [comment]
Ok, thank you for the assistance.

From: José Martins ***@***.***>
Sent: Friday, November 22, 2024 3:39 PM
To: bao-project/bao-hypervisor ***@***.***>
Cc: Pravallika Mittireddy ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [bao-project/bao-hypervisor] Is it possible to restart zephyr os from baremetal side? (Discussion #194)

No. At the moment is not possible to restart a VM. There is a wip PR that allows this. But it only allows a VM can restart itself. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored


No. At the moment is not possible to restart a VM. There is a wip PR<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_bao-2Dproject_bao-2Dhypervisor_pull_157&d=DwMFaQ&c=nKjWec2b6R0mOyPaz7xtfQ&r=UszTRzBdebBAgkRzX3RelxP_-cdTF-xEtDtESyol-kw&m=hzRL9w1So81t6iKrWQg97y6qkIhLP-WBy2xpc7x5aBCV9BEuGoePgZ_IqZOnF1RC&s=M-ZkBLCRfAQdfoiMiKJsUdVGQNgNrxsNg51r-K0bExU&e=> that allows this. But it only allows a VM can restart itself.
